.ocamlformat

```diff
@@ -1,3 +1,3 @@
 module-item-spacing=preserve
-version=0.26.1
```

Comment:
We should do that at some point but in a separate PR

Reply:
Reverted